walletunlocker/service_test.go

```diff
@@ -461,3 +470,16 @@ func TestChangeWalletPassword(t *testing.T) {
 		t.Fatalf("password not received")
 	}
 }
+
+func init() {
```

Comment:
hm, maybe try avoiding using init? Could we move it to `createTestWallet` instead?

Reply:
Yes, good point. Copied this over from the itest PR (#3924) but it can be simpler for unit tests.